services/121-service/src/utils/soap/soap.service.ts

```diff
@@ -146,33 +146,46 @@ export class SoapService {
     return xml;
   }
 
-  async postCBERequest(payload: any, soapAction: string): Promise<any> {
```

Comment:
Maybe out of scope, but isn't it better (following the pattern) that this code gets its own file in `services/121-service/src/payments/fsp-integration/commercial-bank-ethiopia/services/commercial-bank-ethiopia.soap.service.ts`?

Reply:
Agreed but out of scope for this item